pkg/dashboard/static/list-view.js

```diff
@@ -20,14 +20,35 @@ function loadChartsList() {
 }
 
 function buildChartCard(elm) {
-    const card = $(`<div class="row m-0 py-3 bg-white rounded-1 b-shadow border-4 border-start">
-            <div class="col-4 rel-name"><span class="link">release-name</span><div></div></div>
-            <div class="col-3 rel-status"><span></span><div></div></div>
-            <div class="col-2 rel-chart text-nowrap"><span></span><div>Chart Version</div></div>
+    const card = $(`<div class="row m-0 py-4 bg-white rounded-1 b-shadow border-4 border-start">
+            <div class="col-6 rel-name"><span class="link">release-name</span><div></div></div>
+            <div class="col-1 rel-status"><span></span><div></div></div>
+            <div class="col-1 rel-chart text-nowrap"><span></span><div>Chart Version</div></div>
             <div class="col-1 rel-rev"><span>#0</span><div>Revision</div></div>
             <div class="col-1 rel-ns text-nowrap"><span>default</span><div>Namespace</div></div>
             <div class="col-1 rel-date text-nowrap"><span>today</span><div>Updated</div></div>
         </div>`)
+    
+    $.getJSON("/api/helm/repo/search?name=" + elm.name).fail(function (xhr) {
```

Comment:
Do you know if we can avoid this API call? 
Currently I need to first make this call to get the chart name in format `<repo_name>/<chart_name>` and make the second API call (below on line 36). This is the expected format for `helm show` command.

The same API call at https://github.com/komodorio/helm-dashboard/blob/main/pkg/dashboard/static/actions.js#L21 works in 1 go because the input to this is already in the format `<repo_name>/<chart_name>`

Reply:
This creates also creates a problem if the chart is installed with a different name like below. The 1st API call is unable to fetch any results and hence there is no input to make the 2nd PAI call to fetch the icon and description of the chart.
![image](https://user-images.githubusercontent.com/17697790/199952884-8782c9d1-d433-47f8-b361-39795eb283f1.png)


1st API call is `/api/helm/repo/search?name=nginx-test` and the output is null

```DEBU[0003] Starting command: [helm search repo --regexp /nginx-test --versions --output json --kube-context minikube]```

Reply:
Unfortunately, we cannot avoid that call. You can only make it to happen inside Go backend, but the number of Helm calls will remain the same. Unfortunately, I did not find any other way to locate the repo source of a given chart.

We will cache the results of these calls down the road.

Reply:
can we add a hidden element in the HTML that stores the repo name?

Reply:
If you need to store arbitrary piece of information, just use jQuery's `data()` function. It is used widely in the code.